libs/core/src/components/pds-radio/pds-radio.tsx

```diff
@@ -46,6 +46,11 @@ export class PdsRadio {
    */
   @Prop() label: string;
 
+  /**
+   * Visually hides the label text for instances where only the radio should be displayed. Label remains accessible to assistive technology such as screen readers.
+   */
+  @Prop() labelHidden: boolean;
```

Comment:
This is a minor suggestion, but if the team disagrees, please proceed with the merge. Can we change this to read `hideLabel`? When I see `labelHidden`, I think it checks whether it’s hidden.

Reply:
`hideLabel` makes more semantic sense to me. 👍🏽

Reply:
Updating here and will update the other form elements that have this prop